### PR TITLE
Update openfoam plugin

### PR DESCRIPTION
**Pull request by itziakos**

### CODE REVIEW

**Comment by itziakos** [comment]
@nathanfranklin and @SGGgarcia this PR integrates the latest version of simphony-openfoam (0.1.1) to the frame work and bumps the simphony-common version to 0.1.3. This is the first integration on which I used the new Makefile based setup and I think it is much easier to use and test locally. 

Please have a look and let me know it is ok to merge

**Comment by nathanfranklin** [comment]
@itziakos , i won't be able to take a look till later next week.

File: README.rst [hunk 1]
```diff
@@ -12,16 +12,16 @@ library and the simphony plugins.
 Packages
 --------
 
-The simfony version that is supported in version 0.1.2 is:
```
Comment (nathanfranklin) [inline README.rst]: `simfony` -> `SimPhoNy` or `simphony-common`

`version 0.1.2` -> `version 0.1.3`

File: README.rst [hunk 2]
```diff
@@ -12,16 +12,16 @@ library and the simphony plugins.
...
+The simfony version that is supported in version 0.1.2 of the framework is:
 
-- https://github.com/simphony/simphony-common/releases/tag/0.1.1, version 0.1.1
+- https://github.com/simphony/simphony-common/releases/tag/0.1.3, version 0.1.3
 
 The simpony plugins that are compatible with this release:
```
Comment (nathanfranklin) [inline README.rst]: same as comment above:

`simfony` -> `SimPhoNy` or `simphony-common`

**Comment by nathanfranklin** [comment]
left a minor comment related to version number.  looks good.  :+1:   

i also finally used the mayavi simphony plugin. very cool.

File: README.rst (hunk 1 above)
Comment (itziakos) [inline README.rst]: Done

**Comment by itziakos** [comment]
@nathanfranklin, I am inclined to suggest that this PR is modified to not run the openfoam plugin tests, so that we can merge it. However, we need to make sure that the test issue is fixed.

**Comment by nathanfranklin** [comment]
> @nathanfranklin, I am inclined to suggest that this PR is modified to not run the openfoam plugin tests, so that we can merge it. However, we need to make sure that the test issue is fixed.

I agree.

**Comment by itziakos** [comment]
updated code and added issue #35

**Comment by nathanfranklin** [comment]
:+1:

**Comment by itziakos** [comment]
@nathanfranklin this is ready to merge

**Pull request merged**